values/kube-prometheus-stack.yml

```diff
@@ -0,0 +1,97 @@
+## Refs: https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml
+##
+## Default values for kube-prometheus-stack.
+## This is a YAML-formatted file.
+## Declare variables to be passed into your templates.
+##
+
+global:
+  %{~ if pull_secrets == true ~}
+  imagePullSecrets:
+    - name: ${regcred}
+  %{~ endif ~}
+## Using default values from https://github.com/grafana/helm-charts/blob/main/charts/grafana/values.yaml
+##
+grafana:
+  ingress:
+    ## If true, Grafana Ingress will be created
+    ##
+    enabled: true
+    annotations:
+      kubernetes.io/ingress.class: nginx
+      cert-manager.io/cluster-issuer: "${cluster_issuer_name}"
+      acme.cert-manager.io/http01-edit-in-place: "true"
+    hosts:
+      - ${ingress_host}
+    tls:
+      - secretName: ${secret_name}
+        hosts:
+          - ${ingress_host}
+  resources:
+  %{~ if grafana_resources != "" ~}
+    requests:
+      cpu: ${grafana_resources.cpu_requests}
+      memory: ${grafana_resources.memory_requests}
+  %{~ endif ~}
+  persistence:
+    enabled: true
+    %{~ if trimspace(storage_class_name) != "" ~}
+    storageClassName: ${storage_class_name}
+    %{~ endif ~}
+    accessModes: [ "ReadWriteOnce" ]
+    size: ${grafana_pv_size}
+    # annotations: {}
+    finalizers:
+      - kubernetes.io/pvc-protection
+    # selectorLabels: {}
+    # subPath: ""
+    # existingClaim:
+
+    ## If persistence is not enabled, this allows to mount the
+    ## local storage in-memory to improve performance
+    ##
+    inMemory:
+      enabled: false
+      ## The maximum usage on memory medium EmptyDir would be
+      ## the minimum value between the SizeLimit specified
+      ## here and the sum of memory limits of all containers in a pod
+      ##
+      # sizeLimit: 300Mi
+prometheusOperator:
+  %{~ if prometheus_operator_resources != "" ~}
+  resources:
+    requests:
+      cpu: ${prometheus_operator_resources.cpu_requests}
+      memory: ${prometheus_operator_resources.memory_requests}
+  %{~ endif ~}
+
+prometheus:
+  prometheusSpec:
+  %{~ if prometheus_resources != "" ~}
+    resources:
+      requests:
+        cpu: ${prometheus_resources.cpu_requests}
+        memory: ${prometheus_resources.memory_requests}
+  %{~ endif ~}  
+    storageSpec:
+      volumeClaimTemplate:
+        spec:
+        %{~ if trimspace(storage_class_name) != "" ~}
+          storageClassName: ${storage_class_name}
+        %{~ endif ~}
+          accessModes: [ "ReadWriteOnce" ]
+          resources:
+            requests:
+              storage: ${prometheus_pv_size}
+
+# Disable Etcd metrics
+kubeEtcd:
+  enabled: false
+
+# Disable Controller metrics
+kubeControllerManager:
+  enabled: false
+
+# Disable Scheduler metrics
+kubeScheduler:
+  enabled: false
```

Comment:
I think that these configurations should be a variables

Reply:
Done